Tests/GRPCInProcessTransportTests/InProcessServerTransportTests.swift

```diff
@@ -71,7 +71,7 @@ final class InProcessServerTransportTests: XCTestCase {
 
     try transport.acceptStream(firstStream)
 
-    try await transport.listen { stream in
```

Comment:
Aren't we getting a warning because we're not using `context` here? Should we `_` it, or should we assert the contained method descriptor?